tuber/__init__.py

```diff
@@ -41,5 +62,23 @@ def get_root(path: Optional[Union[Path, str]] = None) -> Path:
         path = parent
 
 
+def walk(
+    root: Path,
+    includes: Iterable[str] = DEFAULT_INCLUDES,
+    excludes: Iterable[str] = DEFAULT_EXCLUDES,
+) -> Iterable[str]:
+    """Generator function that calls os.walk on a path, applies includes/excludes, and yields the resulting paths
+
+    walk() uses fnmatch so Unix shell-style wildcards e.g. *, ?, [1-9] are supported. For more information on the wildcards see https://docs.python.org/3/library/fnmatch.html. Note, the patterns in includes/excludes are only applied to the filename itself, not the directories in the path.
```

Comment:
Hmm, I wonder why `make lint` didn't complain about this line length? I can reduce to 88.